### DISCUSSION TITLE
Access Kibana through WebUI

### DISCUSSION THREAD

**Discussion by squares-ui** [Security-Onion-Solutions/securityonion#2822] [Unsupported Versions] [answered]
On a ~fresh SO install, 2.3.21, 36GB Ram. 8 CPU

#### What works
SO main WebUI
"Hunt" section, which is based on Kibana?
Elastic on 9200 works fine from CORS scripting etc

#### But using SO WebUI I click "Kibana" (redirects to https://192.168.1.20/kibana/) I get a 404
"404 page not found"

#### The log /opt/so/log/kibana/kibana.log gives me a fatal
{"type":"log","@timestamp":"2021-02-01T16:48:05Z","tags":["fatal","root"],"pid":292,"message":"TypeError: this.connection.start is not a function\n    at module.exports.internals.Client.internals.Client.start (/usr/share/kibana/node_modules/catbox/lib/client.js:35:27)\n    at caches.forEach (/usr/share/kibana/node_modules/hapi/lib/core.js:323:69)\n    at Map.forEach (<anonymous>)\n    at module.exports.internals.Core._initialize (/usr/share/kibana/node_modules/hapi/lib/core.js:323:25)\n    at module.exports.internals.Core._start (/usr/share/kibana/node_modules/hapi/lib/core.js:236:24)\n    at internals.Server.start (/usr/share/kibana/node_modules/hapi/lib/server.js:501:27)\n    at HttpService.runNotReadyServer (/usr/share/kibana/src/core/server/http/http_service.js:177:31)"}

#### Docker image running
docker ps | grep kibana
9615ec70b3d3   securityonion:5000/securityonion/so-kibana:2.3.21               "/usr/local/bin/so-k…"   7 days ago   Up 5 hours   0.0.0.0:5601->5601/tcp        

#### Docker proxy running
netstat -anlp | grep 443
tcp6       0      0 :::443                  :::*                    LISTEN      8003/docker-proxy

#### Tried other/old URL incase
https://192.168.1.20/app/kibana/
https://192.168.1.20/app/kibana#/

Any pointers welcomed :)

Thanks
Andy

**Comment by Git-Me-Some-Hub** [comment]
I have a very similar issue. I have installed it in the past with no problem on the ESXi 6.7, but I wanted to go to two separate hard drives for OS/NMS. When I do "so-status" it says error for so-elasticsearch, and then kabana does not work.

**Comment by squares-ui** [comment]
No problems for me under status

so-status | egrep "elastic|kibana"
    so-elasticsearch ---------------------------- [ OK ]
    so-kibana ---------------------------- [ OK ]

Other logs look ok, and looked through known issues, but nothing in there for Kibana url?

**Comment by squares-ui** [comment, marked as answer]
rebuilt it
